main.go

```
@@ -2,26 +2,73 @@ package main
+	setupLogger()
+}
+
 func main() {
 	flag.Parse()
-	conf := config.Load(*optConfig)
-	web.SetServerLocation(&conf)
-	results.Initialize(&conf)
-	database.SetDBInfo(&conf)
-	log.Fatal(web.ListenAndServe(&conf))
+	database.SetDBInfo(cfg)
+	web.ListenAndServe(cfg)
```

golangci-lint (1.62.2)

Error return value of `web.ListenAndServe` is not checked

(errcheck)